core/src/HIP/Kokkos_HIP_Parallel_Team.hpp

```diff
@@ -231,7 +237,9 @@ class TeamPolicyInternal<Kokkos::Experimental::HIP, Properties...>
         m_vector_length(0),
         m_team_scratch_size{0, 0},
         m_thread_scratch_size{0, 0},
-        m_chunk_size(::Kokkos::Experimental::Impl::HIPTraits::WarpSize) {}
+        m_chunk_size(::Kokkos::Experimental::Impl::HIPTraits::WarpSize),
+        m_tune_team_size(false),
+        m_tune_vector_length(false) {}
 
   /** \brief  Specify league size, request team size */
   TeamPolicyInternal(const execution_space space_, int league_size_,
```

Comment:
Same general comment as for CUDA we could just have delegating constructors and everything could go to this one.

TeamPolicyInternal(AUTO, AUTO):TeamPolicyInternal(ExecSpace(),-1,-1){} etc.

Reply:
Done